<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1000</width>
      <height>650</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QColumnView" name="columnView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 236);</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButton_1">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>直线</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/res/1.svg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>画笔</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="toolButton_3">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/3.svg</normaloff>
         <normalon>:/res/4.svg</normalon>:/res/3.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>喷桶</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButton_2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/res/2.svg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>曲线</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>直线</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="toolButton_0">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/res/0.svg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>颜色</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="toolButton_4">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/res/5.svg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="toolButton_5">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/res/7.svg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>贝赛尔曲线拟合</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>columnView</zorder>
   <zorder>graphicsView</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_bangzhu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="newone"/>
    <addaction name="open"/>
    <addaction name="resave"/>
    <addaction name="separator"/>
    <addaction name="undo"/>
    <addaction name="redo"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>说明</string>
    </property>
    <addaction name="announce"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>直线拟合轮廓</string>
     </property>
     <addaction name="loweps"/>
     <addaction name="mideps"/>
     <addaction name="higheps"/>
    </widget>
    <addaction name="graygraph"/>
    <addaction name="twocolor"/>
    <addaction name="reversecol"/>
    <addaction name="separator"/>
    <addaction name="getoutline"/>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="rotclockwise"/>
    <addaction name="rotreverseclockwise"/>
    <addaction name="separator"/>
    <addaction name="action3D"/>
    <addaction name="actionrotate"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="frac"/>
    <addaction name="separator"/>
    <addaction name="fontweight"/>
    <addaction name="separator"/>
    <addaction name="rotatecnt"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>绘制</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>茶壶</string>
     </property>
     <addaction name="FrontView"/>
     <addaction name="TopView"/>
     <addaction name="ObliqueView"/>
    </widget>
    <addaction name="menu_6"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_bangzhu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newone"/>
   <addaction name="open"/>
   <addaction name="resave"/>
   <addaction name="separator"/>
   <addaction name="undo"/>
   <addaction name="redo"/>
   <addaction name="separator"/>
   <addaction name="fontweight"/>
   <addaction name="separator"/>
   <addaction name="rotclockwise"/>
   <addaction name="rotreverseclockwise"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newone">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/9.svg</normaloff>:/res/9.svg</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/10.svg</normaloff>:/res/10.svg</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="resave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/8.svg</normaloff>:/res/8.svg</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="announce">
   <property name="text">
    <string>声明</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="undo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/12.svg</normaloff>:/res/12.svg</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/14.svg</normaloff>:/res/14.svg</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="graygraph">
   <property name="text">
    <string>灰度图</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="twocolor">
   <property name="text">
    <string>二值化</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="reversecol">
   <property name="text">
    <string>反色</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="getoutline">
   <property name="text">
    <string>识别轮廓</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="frac">
   <property name="text">
    <string>贝赛尔曲线阶数</string>
   </property>
  </action>
  <action name="fontweight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/16.svg</normaloff>:/res/16.svg</iconset>
   </property>
   <property name="text">
    <string>加粗</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="loweps">
   <property name="text">
    <string>低精度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="mideps">
   <property name="text">
    <string>中精度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="higheps">
   <property name="text">
    <string>高精度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="rotclockwise">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/15.svg</normaloff>:/res/15.svg</iconset>
   </property>
   <property name="text">
    <string>顺时针旋转</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="rotreverseclockwise">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/11.svg</normaloff>:/res/11.svg</iconset>
   </property>
   <property name="text">
    <string>逆时针旋转</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="FrontView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>正视图</string>
   </property>
  </action>
  <action name="TopView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>俯视图</string>
   </property>
  </action>
  <action name="ObliqueView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>斜视图</string>
   </property>
  </action>
  <action name="action3D">
   <property name="text">
    <string>三维显示</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionrotate">
   <property name="text">
    <string>三维旋转</string>
   </property>
  </action>
  <action name="rotatecnt">
   <property name="text">
    <string>旋转次数</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
